### ISSUE TITLE
Is this chat offline and only takes data from ingested data without openAI?

### ISSUE THREAD

**Issue by putuoka** [hillis/hometown-fha-chat#4]
Hi sorry, i'm new to this and want to ask some question.

I'm building a QA web using openAI so far it's working there's a function to fetch data from the web using crawlers, read from text input, and from documents. The one from the document is not finished yet until I found your repo and saw you using ingest while I never knew about ingest. So sorry I'm a newbie and I wonder if this ingest is used to be able to chat offline? because I see you also hosting on vercel for free without limiting users who can access the chat and it's still active for 4 months meaning it's likely not eating a lot of resources to openAI right? Is this ingest the key to saving openAI token prices or can it be offline by itself without openAI (after previously ingesting to openAI first and generating an index file)?

tldr;
can i use your repo and ingest data without openAI key?
